concordium-node/src/p2p/maintenance.rs

```diff
@@ -54,6 +55,10 @@ use std::{
     time::{Duration, Instant},
 };
 
+/// Maximum number of incoming connections to attempt to accept per iteration
+/// of the main connection maintenance loop.
+pub const MAX_NUM_ACCEPT_ATTEMPTS: u32 = 9;
```

Comment:
It would probably be better to make this part of `NodeConfig` (and correspondingly `ConnectionConfig`).

Reply:
Done.